autoload/conoline.vim

```diff
@@ -82,6 +83,12 @@ function! conoline#enable() abort
     autocmd Syntax,ColorScheme * call conoline#enable()
   augroup END
 
+  " Change light/dark when the background changes
+  augroup conoline_monitor_background_setting
+    autocmd!
+    autocmd OptionSet background :if &background == 'light' | execute "ConoLineColorLight" | else | execute "ConoLineColorDark" | endif
```

Comment:
You can drop the `execute`s and just write the ConoLineColor* command names directly. :neckbeard: